gcloud/bigtable/row.py

```diff
@@ -253,3 +324,21 @@ def to_pb(self):
         :returns: The converted current object.
         """
         return data_pb2.RowFilter(cells_per_column_limit_filter=self.num_cells)
+
+
+class StripValueTransformerFilter(_BoolFilter):
```

Comment:
I'm not sure which docs you're referring to, but the goal is to keep the order the same as in [the protobuf spec][1]

[1]: https://github.com/GoogleCloudPlatform/cloud-bigtable-client/blob/03723524694e21ebc14b21b6371d29e316d04130/bigtable-protos/src/main/proto/google/bigtable/v1/bigtable_data.proto#L321-L404